dwa_local_planner/cfg/DWAPlanner.cfg

```diff
@@ -28,6 +28,7 @@ gen.add("forward_point_distance", double_t, 0, "The distance from the center poi
 gen.add("scaling_speed", double_t, 0, "The absolute value of the velocity at which to start scaling the robot's footprint, in m/s", 0.25, 0)
 gen.add("max_forward_inflation", double_t, 0, "The maximum length the robot's footprint is extended forward by", 0.2, 0)
 gen.add("max_sideward_inflation", double_t, 0, "The maximum length the robot's footprint is extended sidewards by", 0.2, 0)
+gen.add("occdist_use_footprint", bool_t, 0, "Use footprint cost when computing the obstacle distance part of the cost function", False)
```

Comment:
`obstacle distance part of the cost function` is this supposed to be `obstacle cost`?

Might be better to rephrase to explain that disabling this means you are evaluating the cost only at the robot pose and not the whole footprint.

Reply:
> obstacle distance part of the cost function is this supposed to be obstacle cost?

Yes, I copied the sentence from the `occdist_scale` parameter:
https://github.com/rapyuta-robotics/ros-navigation/blob/1705c5eb553076abc402262bccc4fb198bdf90b2/dwa_local_planner/cfg/DWAPlanner.cfg#L19.

I can add another sentence to explain what happens when it's false, yes.